From 2b437907d0b7108f790133ba6189cc51652e5972 Mon Sep 17 00:00:00 2001
From: Lin Fuzhen <fuzhen.lin@freescale.com>
Date: Tue, 21 Aug 2012 16:49:11 +0800
Subject: [PATCH 045/310] ENGR00220895-2 Proile: Add power profile support for
 i.MX SOCs

Add two APKs: FSLPowerprofileApp and FSLProfileService for i.MX6 SOCs
These APKs allow end user to tunne system performance.

Signed-off-by: Lin Fuzhen <fuzhen.lin@freescale.com>
---
 imx6/imx6.mk | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/imx6/imx6.mk b/imx6/imx6.mk
index 22a499d..f31fd43 100644
--- a/imx6/imx6.mk
+++ b/imx6/imx6.mk
@@ -18,6 +18,8 @@ PRODUCT_PACKAGES += \
         Camera                                  \
 	FSLOta					\
 	VideoEditor				\
+	FSLProfileApp				\
+	FSLProfileService			\
 	VisualizationWallpapers			\
 	CubeLiveWallpapers			\
 	PinyinIME				\
-- 
1.8.0

